.github/workflows/accuracy-test.yml

```diff
@@ -0,0 +1,50 @@
+name: Accuracy Test
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - "python/sglang/**"
+      - "test/**"
+  pull_request:
+    branches: [ main ]
+    paths:
+      - "python/sglang/**"
+      - "test/**"
+  workflow_dispatch:
+
+concurrency:
+  group: accuracy-test-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  accuracy-test:
+    if: github.repository == 'sgl-project/sglang' || github.event_name == 'pull_request'
+    runs-on: accuracy
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install dependencies
+      run: |
+        source $HOME/venv/bin/activate
+        echo "$HOME/venv/bin" >> $GITHUB_PATH
+
+        pip install --upgrade pip
+        pip install -e "python[all]"
+        pip install flashinfer -i https://flashinfer.ai/whl/cu121/torch2.4/ --force-reinstall
+
+        git clone https://github.com/merrymercy/human-eval.git
+        cd human-eval
+        pip install -e .
+
+    - name: Evaluate MMLU
+      run: |
+        cd test/srt
+        python3 -m unittest test_eval_accuracy_large.TestEvalAccuracyLarge.test_mmlu
+
+    - name: Evaluate HumanEval
```

Comment:
May we include gsm8K? It's sensitive to precision changes.

Reply:
This one is also very sensitive and it is the one that helps us find this bug https://github.com/sgl-project/sglang/pull/888. I have ported gqpa, math, mmlu, humaneval https://github.com/sgl-project/sglang/tree/main/python/sglang/test.
We can port https://github.com/openai/simple-evals/blob/main/mgsm_eval.py later.

Reply:
ok